### ISSUE TITLE
virtual cluster on pai

### ISSUE THREAD

**Issue by RayXu14** [Microsoft/nni#294]
I wonder how to set virtual cluster before running tasks on pai? I do not want to use the default one, is there any option to use non-default virtual cluster?

**Comment by scarlett2018** [comment]
Thanks @RayXu14 for the question, the current release does not support set VC for OpenPAI training service, we will consider adding it in later releases.

**Comment by yds05** [comment]
@RayXu14 , we just released NNI v0.4, which support virtual cluster configuration for OpenPAI. 

You just need to add virtualCluster into nni trial config, like:
```
trial:
  command: python3 mnist.py
  codeDir: ~/nni/examples/trials/mnist
  gpuNum: 1
  cpuNum: 1
  memoryMB: 8196
  image: msranni/nni:v0.4
  virtualCluster: nni
```

**Comment by scarlett2018** [comment]
Closing this item as the feature had released in 0.4, @RayXu14 - please feel free to reopen the issue if you find there are any problems or questions.
